nf_core/lint/files_exist.py

```diff
@@ -95,30 +99,42 @@ def pf(file_path):
 
     # Files that cause an error if they don't exist
     for files in files_fail:
-        if any([os.path.isfile(pf(f)) for f in files]):
+        if any([f in ignore_files for f in files]):
+            continue
+        elif any([os.path.isfile(pf(f)) for f in files]):
```

Comment:
If I'm being picky, you don't need the `elif` as the loop exits after the `continue`. Makes no functional difference though.

```suggestion
        if any([os.path.isfile(pf(f)) for f in files]):
```